airbyte-cdk/python/bin/run-mypy-on-modified-files.sh

```diff
@@ -1,2 +1,3 @@
 set -e
-git diff --name-only --relative --diff-filter=d remotes/origin/master -- . | grep -E '\.py$' | xargs .venv/bin/python -m mypy --config-file mypy.ini --install-types --non-interactive
\ No newline at end of file
+# TODO change this to include unit_tests as well once it's in a good state
+git diff --name-only --relative --diff-filter=d remotes/origin/master -- airbyte_cdk source_declarative_manifest | grep -E '\.py$' | xargs .venv/bin/python -m mypy --config-file mypy.ini --install-types --non-interactive
```

Comment:
We just need to make sure that whenever we will add a new package, we will have to update this. Can we work by exclusion instead? I see that `[tool.mypy] exclude` could do the trick

Reply:
Fine with that too, feel free to update the PR @maxi297 - I can also do it tomorrow morning

Reply:
Switched it to an ignore list instead of an allow list on the git diff level